app/controllers/CalculationController.scala

```diff
@@ -49,16 +49,17 @@ class CalculationController @Inject()(viewLPP: CalculationLPPView,
                                                                                     errorHandler: ErrorHandler,
                                                                                     authorise: AuthPredicate,
                                                                                     controllerComponents: MessagesControllerComponents)
-  extends FrontendController(controllerComponents) with I18nSupport with CurrencyFormatter with FeatureSwitching {
+  extends FrontendController(controllerComponents) with I18nSupport with CurrencyFormatter {
 
   def onPageLoad(penaltyId: String, isAdditional: Boolean): Action[AnyContent] = authorise.async { implicit request =>
-    if (isEnabled(UseAPI1812Model)) {
-      logger.debug(s"[CalculationController][onPageLoad] - Making call to API1812 endpoint")
-      getPenaltyDetailsFromNewAPI(penaltyId, isAdditional)
-    } else {
-      logger.debug(s"[CalculationController][onPageLoad] - Making call to old endpoint")
-      getOldPenaltyData(penaltyId, isAdditional)
-    }
+    logger.debug(s"[CalculationController][onPageLoad] - Making call to old endpoint")
+    getOldPenaltyData(penaltyId, isAdditional)
+  }
+
+  def onPageLoadForNewAPI(penaltyId: String, penaltyCategory: String): Action[AnyContent] = authorise.async { implicit request =>
+    logger.debug(s"[CalculationController][onPageLoadForNewAPI] - Making call to new endpoint")
+    val penaltyCategoryEnum = LPPPenaltyCategoryEnum.find(penaltyCategory).get
+    getPenaltyDetailsFromNewAPI(penaltyId, penaltyCategoryEnum)
```

Comment:
Could you change penaltyId to principalChargeReference to keep things consistent in the method sig and in 'getPenaltyDetailsFromNewAPI'

Reply:
Good spot, I'm surprised that even worked in the first place 🤔

Reply:
Yeah I know, i think as long as the type is correct it doesn't really care too much what the name is